kong/plugins/prometheus/exporter.lua

```diff
@@ -145,6 +147,21 @@ local function init()
                                           {"service", "route", "direction", "workspace"})
   end
 
+  -- AI mode
+  if ai_request then
```

Comment:
this seems always true, should we just remove the `if`?

Reply:
I would propose to keep for visibility and and if one day we are adding a field in `conf` or `license` to enable or disable AI feature.

Reply:
It will be straightforward to guard it with a `conf` or `license` in the future, on the other side, leaving this in our codebase would cause confusion. Let's avoid having this and keep codebase clean.

Reply:
noted, I have just removed it.